spark-frontend/src/screens/Faucet/FaucetVm.tsx

```diff
@@ -73,27 +73,13 @@ class FaucetVM {
       const tx = await tokenContract.mint(accountStore.address, amount);
       await tx.wait();
       notificationStore.toast("Minting successful!", { type: "success" });
+      await this.rootStore.accountStore.addAsset(assetId);
```

Comment:
```suggestion
      await accountStore.addAsset(assetId);
```

У тебя уже есть `accountStore` выше, не нужно к нему обращаться через `rootStore`